components/camel-avro-rpc/camel-avro-rpc-component/src/main/docs/avro-component.adoc

```diff
@@ -277,4 +277,10 @@ Value value filled as array contents.
 
 This component is not supported in Karaf.
 
+== Avro via http SPI
+
+The Avro-rpc component will use the org.apache.camel.component.avro.spi.AvroRpcHttpServerFactory for consumers over http.
+Default implementation leverages artifact org.apache.avro:avro-ipc-jetty.
```

Comment:
```suggestion
The Avro RPC component offers the `org.apache.camel.component.avro.spi.AvroRpcHttpServerFactory` service provider interface (SPI) so that various platforms can provide their own implementation based on their native HTTP server.

The default implementation available in `org.apache.camel:camel-avro-jetty` is based on `org.apache.avro:avro-ipc-jetty`.
```